README.md

```diff
@@ -1 +1,45 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microverse
```

Comment:
- [ ] Nice try on editing the README file. However, kindly try to follow Microverse's [template](https://github.com/microverseinc/readme-template) and try to fill out the sections (please don't leave them empty). README files are important for a professional project 👍🏻